app/components/preprint-record/component.js

```diff
@@ -0,0 +1,27 @@
+import Ember from 'ember';
+/**
+ * Holds preprint records. Requires screenWidth parameter to determine the appropriate title length and avoid multiple
+ * lines. Used by moderation-list component.
+ *
+ * Sample usage:
+ * ```handlebars
+ * {{preprint-record
+ *    model=model
+ *    screenWidth=screenWidth
+ * }}
+ * ```
+ * @class accepted-record
+ **/
+export default Ember.Component.extend({
+    theme: Ember.inject.service(),
+    preprintRecords: Ember.computed('data', 'type', function () {
+        const type = this.get('type');
+        return type == 'accepted' ? this.get('data.acceptedPreprints.records') : type == 'pending' ?
```

Comment:
It would be better to have an object defined with all the possible values and then access the correct key.
We should be using constant values for these variables as well. I think we could just drop them in the config file.
```javascript
{
 key1: value1,
 key2: value2,
}[keyVar];
```